catalog/imaging/imaging-transformer-nitf/src/test/java/org/codice/alliance/transformer/nitf/TreTestUtility.java

```diff
@@ -221,8 +221,8 @@ public static ImageSegment createImageSegment() {
         when(imageSegment.getImageMode()).thenReturn(ImageMode.BANDSEQUENTIAL);
         when(imageSegment.getNumberOfBlocksPerRow()).thenReturn(4);
         when(imageSegment.getNumberOfBlocksPerColumn()).thenReturn(4);
-        when(imageSegment.getNumberOfPixelsPerBlockHorizontal()).thenReturn(512);
-        when(imageSegment.getNumberOfPixelsPerBlockVertical()).thenReturn(512);
+        when(imageSegment.getNumberOfPixelsPerBlockHorizontal()).thenReturn(512l);
```

Comment:
thanks - I misread the "l" as a "1" and couldn't figure out why that would be necessary.  Makes sense now.